tests/Feature/WorkApiTest.php

```diff
@@ -2,24 +2,37 @@
 
 namespace Tests\Feature;
 
+use App\Models\Work;
 use Illuminate\Foundation\Testing\RefreshDatabase;
-use Illuminate\Foundation\Testing\WithFaker;
+use Tests\RefreshSearchIndex;
 use Tests\TestCase;
 
 class WorkApiTest extends TestCase
 {
 
-    use RefreshDatabase;
+    use RefreshDatabase, RefreshSearchIndex;
 
     public function testIndex()
     {
-        $response = $this->get('/api/works');
+        $response = $this->get(route('api.works.index'));
         $response->assertStatus(200);
     }
 
+    public function testSearchByName()
+    {
+        Work::factory()->create(['name' => 'Televízna veža, Bratislava - Kamzík']);
+        Work::factory()->create(['name' => 'Rezidencia Bárdošov, Bratislava']);
+
+        $this->get(route('api.works.index', ['q' => 'kamzik']))
```

Comment:
great to see tests! thanks. also it's now easy to prove, that 

```suggestion
        $this->get(route('api.works.index', ['q' => 'kamz']))
```

wouldn't return anything (until you include asterisks ;) )